test_scripts/API/ATF_SendHapticData.lua

```diff
@@ -0,0 +1,370 @@
+--------------------------------------------------------------------------------
+-- Copyright (c) 2017 Xevo Inc.
+-- All rights reserved.
+--------------------------------------------------------------------------------
+
+Test = require('connecttest')
+local commonFunctions =
+  require('user_modules/shared_testcases/commonFunctions')
+
+--------------------------------------------------------------------------------
+-- TEST BLOCK I : Check normal cases
+--------------------------------------------------------------------------------
+commonFunctions:newTestCasesGroup("Test suite: Check normal cases")
+
```

Comment:
Making this test pass requires changing the preloaded PT manually to allow `SendHapticData`, there is a way to do this automatically with ATF, use the [Alert](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/master/test_scripts/API/ATF_Alert.lua#L101) test cases as an example

Reply:
@jacobkeeler Thank you for your review. Fixed at 9158befab982085f9e0016f6b410711e5be6aa06